src/test/java/github/weichware10/analyse/HeatmapTest.java

```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
      */
     @Test
     public void compHeatmapsShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.CODECHARTS, 4), new ConfigHeatmap());
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 2), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(new TrialData(ToolType.CODECHARTS, "4", "3"), new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(new TrialData(ToolType.EYETRACKING, "2", "3"), new ConfigHeatmap());
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_CODECHARTS_1_EYETRACKING_2.jpg ausgeb",
                 ".../heatmap/COMPHEATMAP_CODECHARTS_4_EYETRACKING_2.jpg",
                 Heatmap.compHeatmaps(hm1, hm2));
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
-        Heatmap hm4 = new Heatmap(new Data(ToolType.CODECHARTS, 6), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigHeatmap());
+        Heatmap hm4 = new Heatmap(new TrialData(ToolType.CODECHARTS, "6", "3"), new ConfigHeatmap());
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg ausgeben",
                 ".../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg",
                 Heatmap.compHeatmaps(hm3, hm4));
 
-        Heatmap hm5 = new Heatmap(new Data(ToolType.EYETRACKING, 8), new ConfigHeatmap());
-        Heatmap hm6 = new Heatmap(new Data(ToolType.ZOOMMAPS, 12), new ConfigHeatmap());
+        Heatmap hm5 = new Heatmap(new TrialData(ToolType.EYETRACKING, "8", "3"), new ConfigHeatmap());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).